test/runtests.sh

```diff
@@ -29,12 +28,17 @@ Options
  -x FX_EXECUTABLE    path to Firefox executable (default: $FX_EXECUTABLE)
  -d                  enable debug logging
  -c                  open JavaScript console and don't quit on completion
+ -t                  run only translator-dependent tests
  TESTS               set of tests to run (default: all)
 DONE
 	exit 1
 }
 
-while getopts "x:dc" opt; do
+DEBUG=false
+UPDATE_TRANSLATORS=true
+TRANSLATOR_TESTS=false
```

Comment:
We don't want to disable translator tests on zotero/zotero, though. We just don't want them to run on Travis until we've mocked external services, and we want automatic translator updating disabled in Travis and on zotero/translators.